lib/Zonemaster/CLI.pm

```diff
@@ -192,7 +192,7 @@ has 'test' => (
     isa           => 'ArrayRef',
     required      => 0,
     documentation => __(
-'Specify test to run case-insensitively. Should be either the name of a module, or the name of a module and the name of a method in that module separated by a "/" character (Example: "Basic/basic01"). This switch can be repeated.'
+'Specify test to run case-insensitively. Should be either the name of a module, the name of a module and the name of a testcase in that module separated by a "/" character (example: "Delegation/delegation01") or the name of a testcase. This switch can be repeated.'
```

Comment:
In other cases in this file, it is written "test case". I think it should be consistent.